package.json

```diff
@@ -1,17 +1,30 @@
 {
-  "author": {
-    "name": "Erik Rose",
-    "url": "https://grinchcentral.com"
+  "name": "fathom",
```

Comment:
We'll have to tweak the name. I lean away from a scoped name—at least one with an org name in it—because history has shown that projects have a tendency to migrate from org to org. Ask anybody still prepending "NS" to Cocoa APIs. :-)